products/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+from .models import Product, Category, Vendor
+from django_extensions.db.models import ActivatorModel
+
+
+class ProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        exclude = ["deactivate_date"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["category"].queryset = Category.objects.filter(status=ActivatorModel.ACTIVE_STATUS)
```

Comment:
This looks good, 
But can we move this in model so when ever we filter always get the active category/products/vendors without checking for status every where.